### DISCUSSION TITLE
Wine 8?

### DISCUSSION THREAD

**Discussion by woj-tek** [Whisky-App/Whisky#221] [General]
Hi,
in https://github.com/IsaacMarovitz/Whisky/issues/210 it was stated that Whisky can't use system wine (why btw?).

I noticed though that Whisky uses Wine 7.7 - is there a reason for this version and not newer 8.x? Is there a plan to upgrade to wine8?

**Comment by IsaacMarovitz** [comment]
Whisky uses GPTK. GPTK is based on CrossOver 22. CrossOver 22 uses Wine 7. Until CX 23 releases, and GPTK updates to it, we are stuck on Wine 7.

**Reply by woj-tek** [reply]
Thanks for clarification! :-)

**Comment by luojunyuan** [comment]
I'm still waiting for gptk1.1 and wine8👀

**Reply by teohhanhui** [reply]
https://www.reddit.com/r/macgaming/comments/16iwtit/whiskys_future/
